temporal/api/cloud/cloudservice/v1/service.proto

```diff
@@ -105,6 +105,29 @@ service CloudService {
         };
     }
 
+    // Failover a multi-region namespace
+    rpc FailoverNamespace (FailoverNamespaceRequest) returns (FailoverNamespaceResponse) {
```

Comment:
Users can trigger a failover at will? Or if this is internal-only API, can it be removed from the user-facing API? Same for `AddNamespaceRegion`.

Reply:
Yes. They are user facing APIs. Customers can add region or failover from UI as well.

Reply:
This is a bit surprising to me that we let users add regions and failover whenever they want but I admit to not knowing enough details about the project.

Reply:
- We have the ability to limit region pairs however we won't stop users from choosing regions that fit their infra needs (where their workers are, other dependent services)
- Users need the ability to trigger a failover for testing (to make sure they have set everything up correctly) and also for a break glass type scenario where they have an outage on their end that Temporal for some reason has not detected